src/Symfony/Component/DependencyInjection/ParameterBag/EnvPlaceholderParameterBag.php

```diff
@@ -42,7 +42,7 @@ public function get($name)
                     return $placeholder; // return first result
                 }
             }
-            if (!preg_match('/^(?:\w++:)*+\w++$/', $env)) {
+            if (!preg_match('/^(?:\w++::?)*+\w++$/', $env)) {
```

Comment:
`if (!preg_match('/^(?:\w*+:)*+\w++$/', $env)) {`

Reply:
Should we allow `env(default::::::BAR)`?